specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,3590 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Machine Learning Workspaces",
+    "description": "These APIs allow end users to operate on Azure Machine Learning Workspace resources.",
+    "version": "2020-03-01"
```

Comment:
The new endpoints already use new version, for old endpoints, I think it should be OK to use old version.

Reply:
Well I left this comment because this is causing LintDiff CI's failure. But currently the pipelines are somehow broken now.
Please let me get to someone fix them first...

Reply:
It turns out to be something wrong with github, the results are not shown in the conversation page. But you can still get the results on `Checks` page.
The CI somehow not reports this now. I think this is OK to me then